src/pyOpenMS/pyopenms/_parquet_query.py

```
@@ -0,0 +1,246 @@
+    """
+
+    def __init__(self, typed_builder=None):
+        self._pending_connector = None
+        self._condition_count = 0
+        self._typed = typed_builder
+        self._typed_has_condition = False
+        if self._typed is None:
+            try:
+                from . import _parquet_filter
+                self._typed = _parquet_filter.ParquetFilterBuilder()
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)